frontend/src/components/ShootEditor.vue

```diff
@@ -354,6 +357,9 @@ export default {
     this.refresh()
   },
   watch: {
+    canPatchShoots (value) {
+      this.$instance.setOption('readOnly', !value)
```

Comment:
Can we briefly discuss an issue I noted down weeks ago?
When the shoot has purpose `INFRA`, we disable all actions for the shoot to prevent the user from accidentally messing around with critical infrastructure even if he has the permission to do so. 
The question is if we should also disable the edit button / make the editor readonly in case the shoot has `INFRA`purpose.  If so, we could include it now. WDYT?

Reply:
yes makes sense I will disable it for `infrastructure` purpose